src/EditorFeatures/CSharpTest/UseIsNullCheck/UseIsNullCheckForEqualityOperatorTests.cs

```diff
@@ -207,7 +207,7 @@ class C
 {
     void M(string s2)
     {
-        if ((s2 is null) is null))
+        if ((object)(s2 is null) is null))
```

Comment:
this seems like a regression.

Reply:
This is a behavior correction, because the old result doesn't compile.
`Cannot convert null to 'bool' because it is a non-nullable value type`

Reply:
💭 It seems like we shouldn't be trying to fix this at all. It should be the responsibility of some other analyzer to replace the whole expression with the value `true`.